knowledge-base/kb-restrictfirstaxis-to-chartpanewidth.md

```diff
@@ -0,0 +1,47 @@
+---
+title: How To Restrict the First Axis Label to be Within the Chart Pane Width
+description: "How to restrict the first axis label to be within the Chart Pane width of CartesianChart element."
+type: how-to
+page_title: How to restrict the first axis Label to be within the Chart Pane width of RadCartesianChart for WPF.
+slug: kb-restrictfirstaxis-to-chartpanewidth
+position: 0
+tags: axis, chart pane, label, cartesianchart
+ticketid: 1620645
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>2023.2.718</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>RadChartView for WPF</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to restrict the first axis label to be within the `Chart Pane` width of `RadCartesianChart` element.
+
+## Solution
+
+Manually clip the horizontal axis based on the plot area of the chart.
```

Comment:
You can one more sentence that tells how the clipping is done. Something like "To do this, set the `Clip` property of the axis."